src/bash/autostartdaemon.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+###
+# Helper script to setup an autostart daemon for openDash
+###
+WorkingDirectory="/home/pi/dash/"
+if [[ $1 != "" ]]
+then
+    WorkingDirectory="/home/pi/$1/dash/"
+fi
+echo ${WorkingDirectory}
+
+#stop and disable dash service
+sudo systemctl stop dash.service
+sudo systemctl disable dash.service
+
+#remove existing opendash service
+sudo systemctl unmask dash.service
+
+sudo bash -c "echo '[Unit]
+Description=Dash
+After=graphical.target
+
+[Service]
+Type=idle
+User=pi
+StandardOutput=inherit
+StandardError=inherit
+Environment=DISPLAY=:0
+Environment=XAUTHORITY=/home/pi/.Xauthority
+WorkingDirectory=${WorkingDirectory}
+ExecStart=/usr/local/bin/ia
+Restart=always
+RestartSec=10s
+KillMode=process
+TimeoutSec=infinity
+
+[Install]
+WantedBy=graphical.target
+' > /etc/systemd/system/dash.service"
+
+sudo systemctl daemon-reload
```

Comment:
Dash shutdown not working
Dash restart not working

Reply:
@rsjudka Any suggestion?

Reply:
hmm imma guess some sort of privilege issue when run via systemd

normally you need to have sudo permissions when running the shutdown command, but there's some config that enables a user logged in via GUI to shutdown, which is why it works fine when running dash manually

maybe check who owns the dash process?